src/resolvers/nodeResolver.ts

```diff
@@ -1,45 +1,117 @@
 import * as Promise from 'bluebird'
 
+import * as _ from 'lodash'
+import * as request from 'request'
+import * as semver from 'semver'
+
+const getAsync = Promise.promisify(request.get, { multiArgs: true })
+
+const BluebirdLRU = require('bluebird-lru-cache')
+
 import { Bundle, FileInfo, Resolver } from '../resolver'
 
+
+// Used below for when no engine version can be determined.
+const DEFAULT_NODE = '0.10.22'
```

Comment:
0.10.22 is based on the default resin-builder needs to be back-compat, alternatively I'd say we need to enforce people specifying it, I was just trying to bring back functionality ASAP